debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/events/TruncateEvent.java

```diff
@@ -0,0 +1,9 @@
+package io.debezium.connector.oracle.logminer.events;
+
+import io.debezium.connector.oracle.logminer.parser.LogMinerDmlEntry;
+
+public class TruncateEvent extends DmlEvent {
```

Comment:
Don't forget to include a `TruncateEventAdapter` and include it in the annotation of `LogMinerEventMarshaller` so that the new class can be serialized successfully by Infinispan when using the Infinispan buffering configuration.

Reply:
I still need to look into that, and also how to test this.